src/client/renderer/Makefile.am

```diff
@@ -69,7 +71,9 @@ librenderer_la_SOURCES = \
 	r_model.c \
 	r_particle.c \
 	r_program.c \
+	r_program_corona.c \
```

Comment:
That's something better suited to master methinks. It's not exactly related to GL2.